library/ssl_tls.c

```diff
@@ -1354,7 +1366,7 @@ int mbedtls_ssl_derive_keys( mbedtls_ssl_context *ssl )
     return( 0 );
 }
 
-#if defined(MBEDTLS_SSL_PROTO_TLS1_3) || defined(MBEDTLS_SSL_PROTO_TLS1_2)
+#if defined(MBEDTLS_SSL_PROTO_TLS1_2)
```

Comment:
I do not see an additional `#endif` being added for this additional `#if defined`, suspicious.

Reply:
it is in 1467 line .

Reply:
Not in this commit.